### ISSUE TITLE
Login form is filled without user authorization

### ISSUE THREAD

**Issue by ghost** [sarahavilov/secure-login#27]
With the old XUL extension you had to manually authorize the autocompletion of the login form, that was its primary goal, to not have a page trick the browser into giving away password credentials without explicit user input.
With the new webextension the login form is always automatically filled, which is the same behaviour you get without the native browser password manager, is there some hidden setting that needs to be enabled?
I'm using secure login 0.1.7 on Firefox 52.0.2.

**Comment by sarahavilov** [comment]
The new add-on does not auto-fill unless otherwise the toolbar button is pressed.

**Comment by ghost** [comment]
Mine does, I have already tried with a clean profile.
Can it be a bad interaction with NoScript?

**Comment by sarahavilov** [comment]
I have no idea. For sure this is not from SL part. Unless the toolbar button is pressed, SL does not inject the script.

**Comment by ghost** [comment]
And yet the login form is auto-filled without me doing anything.
Is there something I can do to debug this issue?

**Comment by sarahavilov** [comment]
Have you tried disabling SL and see if you still have this behavior?

**Comment by ghost** [comment]
Yes, there's no difference in behaviour with or without the extension installed, just the toolbar button with the login form counter.

**Comment by sarahavilov** [comment]
I suggest to create a new profile and see if you can reproduce this on a clean profile without installing any add-on
